addons/l10n_in/views/res_partner_views.xml

```diff
@@ -11,7 +11,11 @@
                 <attribute name="required">l10n_in_gst_treatment in ['regular', 'composition', 'special_economic_zone', 'deemed_export']</attribute>
             </xpath>
             <xpath expr="//field[@name='vat']" position="before">
-                <field name="l10n_in_gst_treatment" invisible="'IN' not in fiscal_country_codes" readonly="parent_id"/>
+                <label for ="l10n_in_gst_treatment"/>
```

Comment:
@jva-odoo I think we have to add invisible to the label otherwise the label will show.what do you think ?
```
<label for ="l10n_in_gst_treatment"  invisible="'IN' not in fiscal_country_codes"/>
 <div class ="o_row" invisible="'IN' not in fiscal_country_codes" >
       <field name="l10n_in_gst_treatment" readonly="parent_id"/>
       <button type="object" name="update_draft_invoices" title="Update Draft Invoices"  class="oe_link fa fa-fw fa-refresh" />
 </div>

Reply:
yes